cmd/build.sh

```diff
@@ -136,6 +136,13 @@ else
     echo "Skipping Go Tests..."
 fi
 
+# run the completion.go file to get the bash completion script
+# To do the string replace first build the script so that we have a consistent name
+go build -gcflags=-trimpath=$GOPATH -asmflags=-trimpath=$GOPATH tools/completion.go
+./completion
```

Comment:
Shall we use a longer placeholder word instead of ./completion to avoid mistakes?

Reply:
Fixed in e513c8c